packages/flutter/lib/src/painting/fractional_offset.dart

```diff
@@ -181,41 +19,29 @@ abstract class FractionalOffsetGeometry {
 /// below the bottom of the rectangle by the height of the [Size].
 ///
 /// The [FractionalOffset] class specifies offsets in terms of a distance from
-/// the top left, regardless of the [TextDirection]. To support both
-/// left-to-right and right-to-left layouts, consider using
-/// [FractionalOffsetDirectional], which is expressed in terms of an offset from
-/// the leading edge, which is then resolved in terms of a [TextDirection]
-/// (typically obtained from the ambient [Directionality]).
-///
-/// A variety of widgets use [FractionalOffset] in their configuration, most
-/// notably:
-///
-///  * [Align] positions a child according to a [FractionalOffset].
-///  * [FractionalTranslation] moves a child according to a [FractionalOffset].
-///
-/// See also:
-///
-///  * [FractionalOffsetDirectional], which (for properties and arguments that
-///    accept the type [FractionalOffsetGeometry]) allows the horizontal
-///    coordinate to be specified in a [TextDirection]-aware manner.
+/// the top left, regardless of the [TextDirection].
```

Comment:
This should point to [Alignment] explicitly, and maybe explain their relationship.

Reply:
Done.

We might want to deprecate FractionalOffset in a later CL.  If we do that, then we can make the relationship clearer.